engine/src/main/java/org/terasology/physics/engine/PhysicsSystem.java

```diff
@@ -146,7 +147,7 @@ public void updateRigidBody(OnChangedComponent event, EntityRef entity) {
 
     @ReceiveEvent(components = {BlockComponent.class})
     public void onBlockAltered(OnChangedBlock event, EntityRef entity) {
-        physics.awakenArea(event.getBlockPosition().toVector3f(), 0.6f);
+        physics.awakenArea(new org.joml.Vector3f(JomlUtil.from(event.getBlockPosition())), 0.6f);
```

Comment:
Unnecessarily qualified class? I see the old `Vector3f` import get removed, so is this needed? Haven't asked my IDE tho